Source/Login/Web/select-provider/IdentityProviderListItem.tsx

```diff
@@ -25,7 +25,8 @@ export class IdentityProviderListItem extends React.Component<IdentityProviderLi
             <DocumentCard type={DocumentCardType.compact} onClick={this.onClick}>
                 <DocumentCardPreview previewImages={[
```

Comment:
Ahh, I can't make suggestions, but here is what we could do to not show the preview if no image is set:
```typescript
                { this.props.provider.imageURL &&
                    <DocumentCardPreview previewImages={[
                        {
                            previewImageSrc: this.props.provider.imageURL,
                            width: 60,
                            height: 60,
                        }
                    ]} />
                }
```

Reply:
Is this the code you're looking for https://github.com/dolittle-platform/Authentication/pull/10/commits/a4ee3159909061b366706f7840bc46737ad3ec2f ?

Reply:
Ohh, yes!